src/Controller/Api/GroupController.php

```diff
@@ -146,17 +187,22 @@ public function postGroupAction(Request $request)
     /**
      * Updates a group.
      *
-     * @ApiDoc(
-     *  requirements={
-     *      {"name"="id", "dataType"="string", "description"="Group identifier"}
-     *  },
-     *  input={"class"="sonata_user_api_form_group", "name"="", "groups"={"sonata_api_write"}},
-     *  output={"class"="Sonata\UserBundle\Model\Group", "groups"={"sonata_api_read"}},
-     *  statusCodes={
-     *      200="Returned when successful",
-     *      400="Returned when an error has occurred while group creation",
-     *      404="Returned when unable to find group"
-     *  }
+     * @Operation(
+     *     tags={"/api/user/groups"},
+     *     summary="Updates a group.",
+     *     @SWG\Response(
+     *         response="200",
+     *         description="Returned when successful",
+     *         @SWG\Schema(ref=@Model(type="Sonata\UserBundle\Model\Group"))
+     *     ),
+     *     @SWG\Response(
+     *         response="400",
+     *         description="Returned when an error has occurred while group creation"
```

Comment:
```suggestion
     *         description="Returned when an error has occurred during the group creation"
```

Also, the summary says "update", did you really mean to write "creation" here?